patches/TerrariaNetCore/Terraria/Terraria.csproj.patch

```diff
@@ -172,16 +124,59 @@
 -		<Message Text="Copying $(OutputName) to Steam Dir..." Importance="high" />
 -		<Copy SourceFiles="$(TargetPath)" DestinationFiles="$(TerrariaSteamPath)\$(OutputName).exe" />
 -		<Copy SourceFiles="$(TargetDir)$(OutputName).pdb" DestinationFolder="$(TerrariaSteamPath)" />
+-	</Target>
++
++	<!-- Import tasks -->
++	<UsingTask TaskName="SynchronizeDirectories" AssemblyFile="$(BuildTasksAssemblyFile)" TaskFactory="$(BuildTasksTaskFactory)" />
++	<UsingTask TaskName="OrganizeReferenceDestinations" AssemblyFile="$(BuildTasksAssemblyFile)" TaskFactory="$(BuildTasksTaskFactory)" />
++
++	<!-- Installs the build's output into the relevant game folder. -->
++	<Target Name="CopyToActualOutputDirectory" AfterTargets="Build">
++		<PropertyGroup>
++			<_ActualOutputDirectory>$(TerrariaSteamPath)</_ActualOutputDirectory>
```

Comment:
Is there any reason to define this property? Can we replace it with `TerrariaSteamPath`?